### PR TITLE
feat: added displayPath support

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✔                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

Add tiles path field to catalog record

### CODE REVIEW

File: src/storageProviders/s3/s3TileStorageProvider.ts [hunk 1]
```diff
@@ -75,6 +75,7 @@ export class S3TileStorageProvider implements IStorageProvider {
     const itemsToDelete = res.Contents?.map((content) => {
       return { Key: content.Key as string };
     });
+    console.log(itemsToDelete)
```
Comment (cl-shaharshavit-rnd) [inline src/storageProviders/s3/s3TileStorageProvider.ts]: remove console

**Review by cl-shaharshavit-rnd — CHANGES_REQUESTED** [review]

File: src/storageProviders/fs/fsTileStorageProvider.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ export class FsTileStorageProvider extends FsStorageProviderBase {
 
   protected parseLocation(discreteArray: IJob<IngestionParams>[]): string[] {
     const directories = discreteArray.map((discrete) => {
-      return path.join(this.fsTilesLocation, discrete.parameters.metadata.productId as string, discrete.parameters.metadata.productType as string);
+      return path.join(this.fsTilesLocation, discrete.parameters.layerRelativePath);
```
Comment (asafMasa) [inline src/storageProviders/fs/fsTileStorageProvider.ts]: fix lint

**Review by asafMasa — CHANGES_REQUESTED** [review]
update mc-models package if it's been used

File: src/storageProviders/fs/fsTileStorageProvider.ts (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/storageProviders/fs/fsTileStorageProvider.ts]: fixed

**Review by asafMasa — APPROVED** [review]

**Review by cl-shaharshavit-rnd — APPROVED** [review]

**Pull request merged**